Dockerfile

```diff
@@ -27,9 +27,7 @@ COPY application/ /data/
 
 # Fix folder permissions
 RUN chown -R www-data:www-data \
-    console/runtime/ \
-    frontend/runtime/ \
-    frontend/web/assets/
```

Comment:
I'd like to see what's either creating those or requiring them, I don't think they're needed anymore...I'll blow everything away again on my side just to make sure it works properly from scratch.